decompressed/base/sbin/random_seed

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+. /rom/usr/sbin/random_seed
+
+mount_root() {
+        if [ -x /usr/sbin/mount_root-mod ]; then
+                /usr/sbin/mount_root-mod
```

Comment:
Questo path è sbagliato, mount_root-mod è in /sbin

Reply:
mh perchè in /sbin ? vorrei tenere tutto in /usr (non cambia nulla però giusto per ordine)

Reply:
perchè i rispettivi di tch sono lì